java/arcs/sdk/Entity.kt

```diff
@@ -21,7 +21,7 @@ abstract class Entity<T> {
     abstract fun schemaHash(): String
     abstract fun decodeEntity(encoded: ByteArray): T?
     abstract fun encodeEntity(): NullTermByteArray
-    abstract fun isSet(): Boolean
+    abstract val isSet: Boolean
```

Comment:
This shouldn't be a property; it will clash with a schema field of the same name.

In fact, `internalId` shouldn't be a property for the same reason.